src/lib/cli/exit.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+
+/**
+ * Internal dependencies
+ */
+
+export function withError( message: string ) {
```

Comment:
That's a clever way! I see that `exit.withError()` is much better than always calling `process.exit()`... Thanks for that! ❤️